frontend/src/components/GrantAccess.vue

```diff
@@ -53,21 +60,15 @@ function onlyDigit(evt: KeyboardEvent) {
 }
 
 async function grantAccess() {
-  const toast = useToast();
   const newUserRoleAssignmentPayload = toRequestPayload(formData.value)
   try {
     await userRoleAssignmentApi.createUserRoleAssignment(newUserRoleAssignmentPayload)
-    toast.success(`User "${newUserRoleAssignmentPayload.user_name}"" is granted with "${formData.value.role.role_name}" access.`)
+    useToast().success(`User "${newUserRoleAssignmentPayload.user_name}"" is granted with "${formData.value.role.role_name}" access.`)
     formData.value = JSON.parse(JSON.stringify(defaultFormData)) // clone default input data.
     router.push('/manage')
   }
   catch (err: any) {
-    console.error("err: ", err)
-    if (err.response.status == 409) {
-      useToast().warning(`Request not completed: ${err.response.data?.detail}`)
-    } else {
-      useToast().error(`Grant Access failed due to an error. Please try again.If the error persists then contact support.\nMessage: ${err.response.data?.detail}`)
-    }
+    return Promise.reject(err)
   }
```

Comment:
This was done to redirect the errors to the errorHandler, because async operations were not being redirected there. The main idea here was to unify all the errors to one single error handler.